src/setup_runs/wrf/read_config_wrf.py

```diff
@@ -19,10 +20,16 @@ class WRFConfig:
     """Project name """
     target: str
     """Target environment for running tasks"""
-    start_date: str
+    start_date: str = field(converter=process_date_string)
     """start first simulation, "%Y-%m-%d %H:%M:%S %Z" or just "%Y-%m-%d %H:%M:%S" for UTC"""
-    end_date: str
+    end_date: str = field(converter=process_date_string)
```

Comment:
```suggestion
    end_date: datetime.time = field(converter=process_date_string)
```

I think the type is actually a datetime (i..e the output of the converter field). Same with the line 2 above